influxql/ast.go

```diff
@@ -1296,14 +1296,27 @@ func (s *SelectStatement) String() string {
 }
 
 // RequiredPrivileges returns the privilege required to execute the SelectStatement.
-func (s *SelectStatement) RequiredPrivileges() ExecutionPrivileges {
-	ep := ExecutionPrivileges{{Admin: false, Name: "", Privilege: ReadPrivilege}}
+// NOTE: Statement must be normalized first (database name(s) in Sources and
```

Comment:
Not sure I understand. If this method is called before the database name as been populated, the result will not be correct.